packages/dropdowns.next/src/types/index.ts

```diff
@@ -250,15 +250,13 @@ export interface IMenuListProps extends HTMLAttributes<HTMLUListElement> {
   zIndex?: IListboxProps['zIndex'];
 }
 
-export interface IMenuButtonProps extends IButtonProps {
-  ref: RefObject<HTMLButtonElement>;
-}
-
 export interface IMenuProps extends HTMLAttributes<HTMLUListElement> {
   /** Appends the menu to the element provided */
   appendToNode?: IMenuListProps['appendToNode'];
   /** Sets the menu button label or renders a provided trigger element */
-  button: string | ((props: IMenuButtonProps) => ReactNode);
+  button: string | ((props: IButtonProps & { ref: RefObject<HTMLButtonElement> }) => ReactNode);
+  /** Provides additional props to the menu button */
```

Comment:
```suggestion
  /** Provides additional props to the menu [Button](/components/button#button) */
```

...for website cross-reference. Compare with https://garden.zendesk.com/components/multiselect#multiselect `renderItem`.